controllers/kubevirtmachine_controller_test.go

```diff
@@ -140,6 +156,8 @@ var _ = Describe("reconcile a kubevirt machine", func() {
 	mockCtrl = gomock.NewController(GinkgoT())
 	workloadClusterMock := workloadclustermock.NewMockWorkloadCluster(mockCtrl)
 	infraClusterMock := infraclustermock.NewMockInfraCluster(mockCtrl)
+	machineFactoryMock := machinemocks.NewMockMachineFactory(mockCtrl)
+	machineMock := machinemocks.NewMockMachineInterface(mockCtrl)
```

Comment:
In addition, these mock are used in 3 tests, while get initiated for all the test. I'm not sure if we need them as a `Describe`global variables, but I don't have a strong opinion about it.

Reply:
It is currently used by 3 tests but can be used for future tests.
Also this doesn't require any sophisticated logic or IO so it has no significance in terms of performance.